mlir/lib/Dialect/LLVMIR/IR/LLVMAttrs.cpp

```diff
@@ -271,10 +271,9 @@ Attribute ConstantRangeAttr::parse(AsmParser &parser, Type odsType) {
       parser.parseInteger(upper) || parser.parseGreater())
     return Attribute{};
   // For some reason, 0 is always parsed as 64-bits, fix that if needed.
```

Comment:
Do we still need that comment now that we are always handling that case?

Reply:
Good catch. Comment updated.